src/main/java/ludum/dare/actors/items/Projectile.java

```diff
@@ -0,0 +1,36 @@
+package ludum.dare.actors.items;
+
+import ludum.dare.actors.GameObject;
+import ludum.dare.actors.player.Player;
+import ludum.dare.components.AnimationComponent;
+import ludum.dare.components.PositionComponent;
+import ludum.dare.components.SizeComponent;
+import ludum.dare.components.upgradeComponents.IRemoveable;
+import ludum.dare.components.upgradeComponents.TimedComponent;
+
+public class Projectile extends GameObject implements IRemoveable {
```

Comment:
This class is not even close to being done.

- [ ] physics body?
- [ ] collide with players or enemies
- [ ] should probably be abstract, so that a fireball projectile can extend this
- [ ] game objects should use the same IRemoveable as we use for Componenets
- [ ] damage component?